mapping-for-navigation/_posts/0000-01-10-circular-junctions.md

```diff
@@ -0,0 +1,199 @@
+-   Turning circle – this is a circular feature with a traversable center point, which allows vehicles to turn around –  **[highway][1108080d]**=**[turning_circle][496a5a09]**
+-   Turning loop – this has the same function as a turning circle but with non-traversable center point –  **[highway][1108080d]**=**[turning_loop][e1236adf]**
+-   Smaller roundabout, but not a mini-roundabout. Treat it the same as a large roundabout, and for consistency put  **[junction][f14df547]**=**[roundabout][bca8053e]** tag on the node of the junction.
+
+[1108080d]: https://wiki.openstreetmap.org/wiki/Key:highway
+
+[e1236adf]: https://wiki.openstreetmap.org/wiki/Tag:highway%3Dturning_loop
+
+[496a5a09]: https://wiki.openstreetmap.org/wiki/Tag:highway%3Dturning_circle
+
+[bca8053e]: https://wiki.openstreetmap.org/wiki/Tag:junction%3Droundabout
+
+[f14df547]: https://wiki.openstreetmap.org/wiki/Key:junction
+
+See examples of mini-roundabouts in the [OpenStreetMap Wiki](https://wiki.openstreetmap.org/wiki/Tag:highway%3Dmini_roundabout#Mini-roundabouts).
+
+### Roundabout
+
+A [roundabout](https://en.wikipedia.org/wiki/Roundabout) is generally circular road [junction](https://wiki.openstreetmap.org/wiki/Junctions) where the traffic inside circle has right of way. These are tagged with **[junction][f14df547]**=**[roundabout](https://wiki.openstreetmap.org/wiki/Tag:junction%3Droundabout)** tag. This tag is only used for road intersections where there is absence of a traffic signal at the entry and exit of roundabout.
+
+[![roundabout][108eb915]][108eb915]
+<sup>_Take a look at the [wireframe view][c134f465] of this roundabout_</sup>
+
+  [108eb915]: ../../images/intersection-modeling/roundabouts/roundabout.png "roundabout"
+  [c134f465]: ../../images/intersection-modeling/roundabouts/roundabout_wireframe.png "wireframe view"
+
+**<a id="how-to-map-roundabout"></a>How to map a roundabout**
+
+To indicate a roundabout — draw a closed line in the direction of the traffic flow, which aligns to Satellite imagery just as one would while tracing a road. Tag the line with a **[junction][f14df547]**=**[roundabout](https://wiki.openstreetmap.org/wiki/Tag:junction%3Droundabout)**  tag and the appropriate **highway**=**\*** tag.
+
+Roundabouts are implied to be one-way roads, but you may specify this explicitly by adding **[oneway](https://wiki.openstreetmap.org/wiki/Key:oneway)**=**[yes](https://wiki.openstreetmap.org/wiki/Tag:oneway%3Dyes)** tag.
+
+Connect all the highways which meets the roundabout based on traffic flows. If a dual carriageway shares traffic with the roundabout, connect each its direction (one arriving, one departing) separately to the respective exit and entrance nodes on the roundabout. Don’t use the same node as entrance to and exit from a roundabout. Each road has to be connected with the roundabout in a separate node.
```

Comment:
@Andygol This is a great point for roundabout modelling when dual carriages join the junction. Can you add a screenshot of what is recommended and what is not recommended here to make it clear with pictures ?

Reply:
@srividyacb I've added screenshots https://github.com/mapbox/mapping/pull/352/commits/46926883fdaeb2b4a46b66197c8feab431356bf5